content/applications/hr/payroll/work_entries.rst

```diff
@@ -0,0 +1,185 @@
+============
+Work Entries
+============
+
+The :guilabel:`Work Entries` dashboard, which can be found by going to :menuselection:`Payroll
+--> Work Entries --> Work Entries`, provides a visual overview of all employee's individual time
+sheets, with each day split into a morning shift and an afternoon shift.
+
+.. image:: work_entries/work-entries-overview.png
+   :align: center
+   :alt: Work Entries dashboard view showing all employee's work entries.
+
+To change the view so only the entries for a single day, a week, or a whole month are shown, click
+on either :guilabel:`Day`, :guilabel:`Week`, or :guilabel:`Month` to view only those entries.
```

Comment:
```suggestion
To change the view so that only the entries for a single day, week, or month are shown, click
on either :guilabel:`Day`, :guilabel:`Week`, or :guilabel:`Month`.
```